### PR TITLE
chore: Add ci format code

### PR DESCRIPTION
**Pull request by RetricSu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,14 @@
     "packages/*"
   ],
   "scripts": {
+    "fmt": "yarn workspace @polyjuice-provider/godwoken fmt && yarn workspace @polyjuice-provider/base fmt && yarn workspace @polyjuice-provider/ethers fmt && yarn workspace @polyjuice-provider/web3 fmt",
```
Comment (classicalliu) [inline package.json:11]: Seems equal to `yarn workspaces run fmt`

File: package.json [hunk 2]
```diff
@@ -8,7 +8,14 @@
     "packages/*"
   ],
   "scripts": {
+    "fmt": "yarn workspace @polyjuice-provider/godwoken fmt && yarn workspace @polyjuice-provider/base fmt && yarn workspace @polyjuice-provider/ethers fmt && yarn workspace @polyjuice-provider/web3 fmt", 
+    "lint": "yarn workspace @polyjuice-provider/godwoken lint && yarn workspace @polyjuice-provider/base lint && yarn workspace @polyjuice-provider/ethers lint && yarn workspace @polyjuice-provider/web3 lint",
```
Comment (classicalliu) [inline package.json:12]: Also here for `yarn workspaces run lint`

**Review by classicalliu — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (RetricSu) [reply package.json:11]:
```suggestion
    "fmt": "yarn workspaces run fmt", 
```

File: package.json (hunk 2 above)
Comment (RetricSu) [reply package.json:12]:
```suggestion
    "lint": "yarn workspaces run lint",
```

**Pull request merged**